controls/curl_works.rb

```diff
@@ -1,7 +1,5 @@
-curl_test = attribute('curl_test', default: '/bin/curl Sleeping /hab/svc/curl/hooks/run')
+curl_test = attribute('curl_test', default: '/bin/curl --version')
```

Comment:
To avoid deprecation warnings, this might be better as:
```
 curl_test = input('curl_test', value: '/bin/curl --version') 
```

Reply:
changed to inputs in both curl_works and curl_exists tests